packages/drivers/odsp-driver/src/fetch.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    default as nodeFetch,
+    RequestInfo as FetchRequestInfo,
+    RequestInit as FetchRequestInit,
+} from "node-fetch";
+
+// The only purpose of this helper is to work around the slight misalignments between the
+// Browser's fetch API and the 'node-fetch' package.
+//
+// Originally, our code simply omitted the '@types/node-fetch' dependency and let TypeScript
+// use the 'fetch' declarations in 'DOM.lib.ts'.  This works until some other package introduces
+// a dependency on '@types/node-fetch' and Lerna hoists '@types/node-fetch' to the root /node_modules/.
```

Comment:
I think it's worth mentioning explicitly that the input/output types are from lib.dom. It probably seems uber obvious to you but as someone who hasn't spent much time thinking in this space, it took me a minute to get what was going on here.

And I don't think we need the "Originally..." comment, this adapter makes sense and for context people can find this PR.

Comment could be something like this (please proofread!):

```suggestion
// The only purpose of this helper is to work around the slight misalignments between the
// Browser's fetch API and the 'node-fetch' package, by wrapping the call to the 'node-fetch' API
// in the browser's types from lib.dom.d.ts (which we use throughout the codebase).
```

Reply:
Oops.  Sorry, I didn't scroll down enough before merging.  I'll bundle the refined comment w/my next change.